extensions/amp-video/0.1/amp-video.js

```diff
@@ -29,8 +29,14 @@ import {assertHttpsUrl} from '../../../src/url';
 const TAG = 'amp-video';
 
 /** @private {!Array<string>} */
-const ATTRS_TO_PROPAGATE_ON_BUILD = ['poster', 'controls', 'aria-label',
-  'aria-describedby', 'aria-labelledby'];
+const ATTRS_TO_PROPAGATE_ON_BUILD = [
+  'aria-describedby',
+  'aria-label',
+  'aria-labelledby',
+  'controls',
+  'crossorigin',
```

Comment:
let's document this in the `md` file and mention the need for it when `track` is used from `crossorigin` sites.